.changeset/cold-cheetahs-double.md

```diff
@@ -0,0 +1,12 @@
+---
+'@directus/storage-driver-cloudinary': patch
```

Comment:
Just a note: I was thinking this could be a breaking change due to the read function signature changing but i dont think we've ever exposed these drivers to extensions/devs